app/src/main/java/com/github/se/bootcamp/ui/screens/SettingsScreen.kt

```diff
@@ -0,0 +1,143 @@
+package com.github.se.bootcamp.ui.screens
+
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material.icons.outlined.ArrowBack
+import androidx.compose.material.icons.outlined.Edit
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.github.se.bootcamp.ui.navigation.NavigationActions
+
+@Composable
+fun SettingsScreen(profilePictureUrl: String?,
+                   navigationActions: NavigationActions) {
+    var username by remember { mutableStateOf("Test Name 1") }
+    var showEditOptions by remember { mutableStateOf(false) }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            //.background(Color.DarkGray)
```

Comment:
Ideally, we should avoid merging commented code into the main branch. Commented code tends to stay longer than expected and can accumulate technical debt. If this code is necessary for future work, it might be better to track it in a task or discussion, or remove it until it's ready for use.